esbuild.mjs

```diff
@@ -5,7 +5,7 @@ const watch = args.includes('--watch')
 
 await build({
   bundle: true,
-  entryPoints: ['./app/email-incoming/index.js'],
+  entryPoints: ['./app/email-incoming/index.ts'],
```

Comment:
What does this change do?

Reply:
the actual file is a ts file, not js

Reply:
https://github.com/nasa-gcn/gcn.nasa.gov/blob/main/app/email-incoming/index.ts

Reply:
OK. It would be better next time for this kind of change to be in a separate PR. (Keep each PR focused on one set of related changes.)